invenio_query_parser/utils.py

```diff
@@ -68,17 +68,33 @@ def generate_keywords():
         from invenio_base.globals import cfg
         keyword_mapping = cfg['SEARCH_ELASTIC_KEYWORD_MAPPING']
         json_schema_paths = cfg['JSON_SCHEMA_PATHS']
+        elastic_config_paths = cfg['ELASTIC_MAPPINGS_PATHS']
```

Comment:
These 3 variables should be defined with empty defaults in the `config.py` file of invenio-query-parser.